dirs.proj

```diff
@@ -0,0 +1,6 @@
+<Project Sdk="Microsoft.Build.Traversal">
```

Comment:
@weshaggard what does this do? I thought we were doing multiple directory.build.props files for providing common references